src/FeaturesViewer.cpp

```diff
@@ -97,7 +97,7 @@ void FeaturesViewer::reload()
     }
 }
 
-void FeaturesViewer::onResultReady(QList<Feature*> features)
+void FeaturesViewer::onResultReady(const QList<Feature*>& features)
```

Comment:
In theory, this would be dangerous since we receive the list of features from another thread.
In practice, Qt always copies the arguments for non direct signal connections, so it's safe. But IMHO, assuming the fact that there is a copy here is more explicit.
See https://stackoverflow.com/a/18146433

Reply:
ok i'll roll it back